service/src/main/java/tds/session/services/impl/SessionServiceImpl.java

```diff
@@ -23,4 +23,11 @@ public SessionServiceImpl(SessionRepository sessionRepository) {
     public Optional<Session> getSessionById(UUID id) {
         return sessionRepository.getSessionById(id);
     }
+
+    @Override
+    public void pause(final UUID sessionId, final String newStatus) {
+        // TODO:  Throw an exception if attempting to pause a Session that is already paused?
```

Comment:
@jeffjohnson9046 and I discussed this offline.  We're going to create a JIRA to handle the exam side of the pause after we have the exam start completed.